scripts/export_import_between_pageservers.py

```diff
@@ -450,13 +450,13 @@ def add_missing_rels(base_tar, output_tar, log_dir, pg_bin, tmp_pg_port: int):
 def get_rlsn(pageserver_connstr, tenant_id, timeline_id):
     conn = psycopg2.connect(pageserver_connstr)
     conn.autocommit = True
-    with conn.cursor() as cur:
+    with closing(conn.cursor()) as cur:
```

Comment:
That should work without `closing`, there are plenty of other places like this. Would you mind double-checking?

Reply:
```python
with psycopg2.connect(pageserver_connstr) as conn:
  conn.autocommit = True
  with conn.cursor() as cur:
    ...
```

I think it should be sth like this